### PR TITLE
Fix urlcam.py to retreive scan images from the PiZeroCam

### LINKED ISSUE #3: Fix `Camera` class from `urlcam`.
Receive an error about `io.BytesIO`

### PR DESCRIPTION
**Pull request by jlegrand62**
Close #3

### CODE REVIEW

File: romiscanner/urlcam.py [hunk 1]
```diff
@@ -21,28 +21,47 @@
...
+    def grab(self, idx: int, metadata: dict = None):
         data_item = hal.DataItem(idx, metadata)
-        data = imageio.imread(BytesIO(requests.get(self.url).content))
+        # https://docs.python.org/3/library/http.server.html#http.server.BaseHTTPRequestHandler.wfile
+        # wfile:
+        #   Contains the output stream for writing a response back to the client.
+        #   Proper adherence to the HTTP protocol must be used when writing to this stream in order to achieve successful interoperation with HTTP clients.
+        #   Changed in version 3.6: This is an io.BufferedIOBase stream.
+        # data = imageio.imread(BytesIO(requests.get(self.url+"scan.jpg").content))
+        _ = requests.get(self.url + "/scan.jpg")  # update the picture
```
Comment (jlegrand62) [inline romiscanner/urlcam.py:64]: Make a `get` request to update the image by calling `grab` from the `picamera` module in `picamera_server.py`.

Comment (jlegrand62) [reply romiscanner/urlcam.py:64]: Code for `picamera_server.py` can be found on the ROMI documentation [here](https://docs.romi-project.eu/Scanner/build/picamera_setup/#camera-serve-python-code).

File: romiscanner/urlcam.py [hunk 2]
```diff
@@ -21,28 +21,47 @@
...
-        data = imageio.imread(BytesIO(requests.get(self.url).content))
-        data_item.add_channel("rgb", data)
+        # https://docs.python.org/3/library/http.server.html#http.server.BaseHTTPRequestHandler.wfile
+        # wfile:
+        #   Contains the output stream for writing a response back to the client.
+        #   Proper adherence to the HTTP protocol must be used when writing to this stream in order to achieve successful interoperation with HTTP clients.
+        #   Changed in version 3.6: This is an io.BufferedIOBase stream.
+        # data = imageio.imread(BytesIO(requests.get(self.url+"scan.jpg").content))
+        _ = requests.get(self.url + "/scan.jpg")  # update the picture
+        data = imageio.imread(self.url + "/scan.jpg")  # download it
```
Comment (hanappe) [inline romiscanner/urlcam.py:65]: Could you try line 65 only, without the preliminary call in line 64?

**Review by hanappe — CHANGES_REQUESTED** [review]
See comment on line 65

**Pull request merged**